lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -284,6 +301,44 @@ class LighthouseViewerReport {
   }
 
   /**
+   * Handles changes to the gist url input.
+   */
+  onInputChange(e) {
```

Comment:
what does this look like on mobile then after entering the url, keyboard prompts done? should we add our own button?